src/cache/cache_las.c

```diff
@@ -206,12 +206,10 @@ __wt_las_cursor_open(WT_SESSION_IMPL *session)
 	 * opens (the first update is safe because it's single-threaded from
 	 * wiredtiger_open).
 	 */
-	if (!F_ISSET(btree, WT_BTREE_LOOKASIDE))
-		F_SET(btree, WT_BTREE_LOOKASIDE);
-	if (!F_ISSET(btree, WT_BTREE_NO_CHECKPOINT))
-		F_SET(btree, WT_BTREE_NO_CHECKPOINT);
-	if (!F_ISSET(btree, WT_BTREE_NO_LOGGING))
-		F_SET(btree, WT_BTREE_NO_LOGGING);
+#define	WT_BTREE_LAS_FLAGS						\
+	(WT_BTREE_LOOKASIDE | WT_BTREE_NO_CHECKPOINT | WT_BTREE_NO_LOGGING)
+	if (F_MASK(btree, WT_BTREE_LAS_FLAGS) != WT_BTREE_LAS_FLAGS)
```

Comment:
Why do we care about checking first?
I also thought we were moving away from LAS as an abbreviation?